src/servers/middlewares/auth-router.ts

```diff
@@ -114,4 +120,129 @@ authRouter.post(
   }),
 )
 
+authRouter.post(
+  "/login",
+  wrapAsyncToRunInSpan({
+    namespace: "auth-router",
```

Comment:
namespaces should include layers structure. so this should be more like `servers.middlewares.authRouter` and please add `fnName` with `login`